CleverTapSDK/InApps/CTInAppEvaluationManager.m

```diff
@@ -215,6 +279,25 @@ - (void)removeSentSuppressedClientSideInApps:(NSDictionary *)header {
     }
 }
 
+- (void)removeSentEvaluatedServerSideInAppIdsForProfile:(NSDictionary *)header {
+    NSArray *inapps_eval = header[CLTAP_INAPP_SS_EVAL_STORAGE_KEY_PROFILE];
```

Comment:
The header key is the same for profile and event queues, it should be `CLTAP_INAPP_SS_EVAL_META_KEY`.